cargo-pgrx/Cargo.toml

```diff
@@ -25,19 +25,19 @@ edition = "2021"
 
 [dependencies]
 atty = "0.2.14"
-cargo_metadata = "0.15.4"
+cargo_metadata = "=0.15.4" # locked b/c we're using an older `clap`
```

Comment:
we have a bunch of different brass instruments but no trombone emoji... sad.

Reply:
I own 3, Jubilee!  Plus a trumpet and a bugle.

Reply:
![image](https://github.com/pgcentralfoundation/pgrx/assets/5117238/3c6eae95-880f-4803-95a6-4c8602d528fb)